packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -74,17 +75,20 @@ export class StatsAnalyzer extends EventsScope {
   statsInterval: NodeJS.Timeout;
   statsResults: any;
   statsStarted: any;
+  receiveSlotCallback: (csi: number) => ReceiveSlot;
 
   /**
    * Creates a new instance of StatsAnalyzer
    * @constructor
    * @public
    * @param {Object} config SDK Configuration Object
+   * @param {Function} receiveSlotCallback Callback used to access receive slots.
    * @param {Object} networkQualityMonitor class for assessing network characteristics (jitter, packetLoss, latency)
    * @param {Object} statsResults Default properties for stats
    */
   constructor(
     config: any,
+    receiveSlotCallback: (csi: number) => ReceiveSlot = () => undefined,
```

Comment:
the return type should be `ReceiveSlot | undefined`, shouldn't it? I'm surprised typescript is not complaining when you're assigning it to `() => undefined`

Reply:
Right, updated the type.

Reply:
I see you updated it in line 78, but here it's defined again and without the `I undefined` - probably best to define an actual type and use it in both places